lib/shoryuken/manager.rb

```diff
@@ -13,22 +12,22 @@ def initialize(fetcher, polling_strategy)
       @queues = Shoryuken.queues.dup.uniq
 
       @done = Concurrent::AtomicBoolean.new(false)
-      @dispatching = Concurrent::AtomicBoolean.new(false)
 
       @fetcher = fetcher
       @polling_strategy = polling_strategy
 
-      @heartbeat = Concurrent::TimerTask.new(run_now: true,
```

Comment:
@waynerobinson I'm doing the `loop` in a `ensure` block, and also calling `dispatch_async` when a processor done, just in case. So I don't think we need to the heartbeat anymore.